
From: Jens Axboe <axboe@suse.de>

It's just a typo (well too much copy'n paste, actually), this should fix
it.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/md/md.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/md/md.c~barrier-md-fix drivers/md/md.c
--- 25/drivers/md/md.c~barrier-md-fix	Mon Aug  2 16:08:48 2004
+++ 25-akpm/drivers/md/md.c	Mon Aug  2 16:08:48 2004
@@ -167,7 +167,7 @@ int md_flush_mddev(mddev_t *mddev, secto
 		request_queue_t *r_queue = bdev_get_queue(rdev->bdev);
 		int err;
 
-		if (!r_queue->unplug_fn)
+		if (!r_queue->issue_flush_fn)
 			err = -EOPNOTSUPP;
 		else
 			err = r_queue->issue_flush_fn(r_queue, rdev->bdev->bd_disk, error_sector);
_
